components/table/stories/table.stories.js

```diff
@@ -242,22 +425,58 @@ Quiet.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * Tables with enabled column sorting can show different states of sorting: unsorted, ascending, and descending. Additionally, tables can trigger a menu, as indicated by the chevron. The `aria-sort` attribute should be set on the `.spectrum-Table-headCell` element for the column that is currently sorted. `aria-sort` should not be set on more than one column at a time.
+ *
+ * When a column is sorted in ascending order, use the `SortUp` icon, as seen below. If sorted in descending order, use the `SortDown` icon. If the sort is undefined, use the more general `Sort` icon.
+ *
+ * Implementations will develop their own sorting functionality. The `aria-sort` attribute should be removed from a column header when the column is not sorted.
+ */
+export const Sortable = Template.bind({});
+Sortable.args = {
+	...Default.args,
+	isSortable: true,
+	sortIcon: "Sort",
+};
+Sortable.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+Sortable.storyName = "Sortable columns";
+Sortable.tags = ["!dev"];
+
+/**
+ * Tables can have additional actions in the header cells that could trigger a menu. The content of the header cell is an [button](/docs/components-button--docs). Oftentimes, the menu is used to show users the available sorting options.
```

Comment:
Implementation looks good! Just a couple suggestions:
1. Can this be re-ordered so it is the story right after sortable columns?
2. I think we should demonstrate in the markup - and mention - that `aria-sort` should still be used when sorting is active
3. It looks like an extra class of `spectrum-Table-tableButton` is needed on the button, which I would elevate into both descriptions.
4. Also note the extra class of `spectrum-Table-menuButton` in this description
5. Additional slight wording update as suggested (feel free to edit further)

```suggestion
 * Tables can have additional actions in the header cells that could trigger a menu. Oftentimes, the menu is used to show users the available sorting options.  The content of the header cell is an [button](/docs/components-button--docs), which should contain the sorting icon when applicable as well as the header label.
```

Reply:
> Can this be re-ordered so it is the story right after sortable columns?

✅ YES! 

> I think we should demonstrate in the markup - and mention - that aria-sort should still be used when sorting is active

:white Let me know if this still needs tweaking or improvement. Is this what you were thinking?
"The `aria-sort` attribute should be set on the `.spectrum-Table-headCell .is-sortable` element for the column that is currently sorted, and when sorting is active."
"The `aria-sort` attribute should be set on the `.spectrum-Table-hasMenuButton` element for the column that is currently sorted, and when sorting is active."

> It looks like an extra class of spectrum-Table-tableButton is needed on the button, which I would elevate into both descriptions.

I did adjust this slightly. The classes have been updated from `.spectrum-Table-menuButton` to `.spectrum-Table-hasMenuButton`, as well as `.spectrum-Table-tableButton` to `.spectrum-Table-sortButton`. As I was working through the suggestions, they were tripping me up, so hopefully that clears up the CSS and the documentation in one go.

I mention `.spectrum-Table-sortButton` in both descriptions, and believe that class is being added to the button appropriately in the markup. Let me know if you're not seeing that.

> Also note the extra class of spectrum-Table-menuButton in this description

✅ mentioned `.spectrum-Table-hasMenuButton`

> Additional slight wording update as suggested (feel free to edit further)
✅

Reply:
LGTM!